models/eval/inspirational_generation.py

```diff
@@ -200,11 +201,16 @@ def gradientDescentOnInput(model,
     gradientDecay = 0.1
 
     nImages = input.size(0)
+    assert nImages == 1
```

Comment:
why always 1?

Reply:
For not mixing the different multiobjective functions.
Anyway the original code, I believe, is not good for the case nImages > 1..